inductiva/_cli/cmd_tasks/list.py

```diff
@@ -19,15 +19,38 @@ def list_tasks(args):
 
     table_dict = tasks.to_dict(task_list)
 
+    def color_formater(status):
+        if status == "success":
+            return utils.format_utils.emphasis_formater(status, "green")
+        elif status in ["failed", "killed", "executer-failed", "zombie"]:
+            return utils.format_utils.emphasis_formater(status, "red")
+        return status
+
     formatters = {
-        "Submitted": utils.format_utils.datetime_formatter,
-        "Started": utils.format_utils.datetime_formatter
+        "Submitted": [
+            utils.format_utils.datetime_formatter,
+            utils.format_utils.spacing_formater
+        ],
+        "Started": [
+            utils.format_utils.datetime_formatter,
+            utils.format_utils.spacing_formater
+        ],
+        "Status": [utils.format_utils.spacing_formater],
+        "ID": [utils.format_utils.spacing_formater],
+        "Simulator": [utils.format_utils.spacing_formater],
+        "Computation Time": [utils.format_utils.spacing_formater],
     }
 
-    print(utils.format_utils.get_tabular_str(
-        table_dict,
-        formatters=formatters,
-    ))
+    header_formatters = [lambda x: x.upper()]
+    if not utils.format_utils.getenv_bool("DISABLE_TERMINAL_EMPHASIS", False):
```

Comment:
@sergiomsantos I think we can avoid another env variable here, since this is a matter of our style. Or do you think in some terminals this can be bugged and people should turn it off?

We maybe should start adding all the env variables into a document/task to keep track of them before we document them properly.